scripts/rmc_flags.py

```diff
@@ -179,6 +179,16 @@ def add_visualizer_flags(make_group, add_flag, config):
                   this is generally the location from which rmc is currently being invoked
                   """)
 
+# Add flags needed for toggling and switching between outputs.
+def add_output_flags(make_group, add_flag, config):
+    group = make_group("Output flags", "Toggle between different styles of output")
+    add_flag(group, "--newOutput", default=False, action=BooleanOptionalAction,
+             help="Turn on new output")    
+    add_flag(group, "--oldOutput", default=True, action=BooleanOptionalAction,
```

Comment:
the name convention is to separate words with '-'. So this should be `--old-output`.